createAnimatableComponent.js

```diff
@@ -389,21 +390,24 @@ export default function createAnimatableComponent(WrappedComponent) {
       if (reversed) {
         easing = Easing.out(easing);
       }
-
-      Animated.timing(animationValue, {
+      const delay = iterationPause ? { delay: iterationPause } : {}
```

Comment:
Not a fan of this style, can't you just do it inline with a default to `0` like the others?